src/PortingAssistant.Client.Client/FileParser/GitConfigFileParser.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.IO;
+using LibGit2Sharp;
+
+namespace PortingAssistant.Client.Client.FileParser
+{
+    public class GitConfigFileParser
+    {
+        public static string getGitRepositoryRootPath(string solutionPath)
+        {
+            string dirName = Path.GetDirectoryName(solutionPath);
+            if (dirName == null)
```

Comment:
use String.IsNullOrWhiteSpace(dirName) instead of just checking nulls

Reply:
will update it